BrokerLock/BrokerLock.h

```diff
@@ -0,0 +1,33 @@
+/**@file
+ * This file is part of the TASTE Runtime Common.
+ *
+ * @copyright 2021 N7 Space Sp. z o.o.
+ *
+ * TASTE Runtime Common was developed under a programme of,
+ * and funded by, the European Space Agency (the "ESA").
+ *
+ * Licensed under the ESA Public License (ESA-PL) Permissive,
```

Comment:
GPL. Maybe just better copy the header from Perfmon.c.

Reply:
ack